docs/proposals/observability-management.md

```diff
@@ -0,0 +1,82 @@
+* [Observability Management](#observability-management)
+  * [Motivation](#motivation)
+    * [Goals](#goals)
+  * [Proposal](#proposal)
+  * [Design Details](#design-details)
+    * [Monitoring Metrics](#monitoring-metrics)
+    * [Collecting Logs](#collecting-logs)
+    * [Display](#display)
+  * [Key Deliverable](#key-deliverable)
+  * [Road Map](#roadmap)
+
+# Observability Management
+
+## Motivation
+Currently, users can only check the status, parameters and metrics of tasks via the command line after creating edge-cloud synergy AI tasks by sedna.
+
+This proposal provides observability management for displaying logs and metrics to monitor tasks in real time, so that users can easily check the status, parameters and metrics of tasks.
+
+### Goals
+* The status of edge nodes and cloud nodes can be collected.
+* The metrics and status of Sedna's components, such as Global Manager, Local Controllers and Workers, can be monitored.
+* The parameters of edge-cloud synergy AI tasks like the count of inference can be collected and displayed on the Observability management.
+* Logs of all pods created by Sedna can be collected to manage and display.
+* Observability data collected can be displayed on Grafana appropriately and aesthetically.
+
+## Proposal
+We propose using Prometheus and Loki to collect observability data like metrics and logs. 
+And the Observability data can be displayed with functions of Grafana.
+
+## Design Details
+![](./images/observability-management-architecture.png)
+
+### Monitoring Metrics
```

Comment:
There are no common system indicators and algorithm indicators for evaluating the performance of the paradigms.

Reply:
The metrics have been added in the new version